### ISSUE TITLE
Match zoom and/or location in multiple images

### ISSUE THREAD

**Issue by porg** [jurplel/qView#458]
## Best Practises
Photoshop has a feature to open multiple images in different arrangements (i.e. tiled next to each other) and then sync the zoom factor and panning between the windows/tiles.

- As can be read in this [Adobe Photoshop support document](https://helpx.adobe.com/photoshop/using/viewing-multiple-images.html)
- As can be seen in action in this [Youtube Video](https://youtu.be/vh2LOEG9Zf8?t=3m23s), see still image here:

![Match zoom and location in multiple images in Photoshop](https://user-images.githubusercontent.com/737143/141520529-26282e13-79fd-431e-8dff-d233dd9dc3ed.png)

## Feature Request
Would be cool if qView could have such a feature to sync zoom and/or location.

## Design Proposal
- In a first version a global option to apply this for all open windows
- And in a later more refined feature "Only in flagged windows", that are windows which have one of the flags "Sync zoom" and/or "Sync panning" activated.

Menu > View >
 √ Sync zoom
 √ Sync pan
 √ only in flagged windows

### Behavior
- "Sync zoom" and "Sync pan" off
  -->  Feature not active anywhere.
- Any of "Sync zoom" or "Sync pan" ON and "only in flagged windows" OFF
  -->  Feature(s) globally on
- Any of "Sync zoom" or "Sync pan" ON and "only in flagged windows" ON
  -->  Per each image window: In the windows bar left of the file icon two icons appear in inactive color:
  - [Magnifier glass](https://fontawesome.com/icons/search) — Hovering over it shows "Sync zoom".
  - [CSS cursor all-scroll](https://developer.mozilla.org/en-US/docs/Web/CSS/cursor#:~:text=all-scroll) — Hovering over it shows "Sync pan".

**Comment by jurplel** [comment]
That would be super cool. I think the easiest implementation would only make sense on mac though, as all windows are separate applications outside of macOS.

**Comment by porg** [comment]
Yes, let's make that a Mac only feature! Please assign that label, I cannot do that.

(From a usability point I always welcomed the Apple Human interface guidelines that enforced "one app instance only for GUI apps". Multiple instances of the same app is something for rather tech-savvy users. This is yet another instance where the benefits of this design guideline can be seen.)

**Comment by jurplel** [comment]
This functionality is pretty advanced, so I will maybe consider later but currently qView has greater regressions that need addressed.

**Comment by porg** [comment]
Totally agree!

**Comment by jurplel** [comment]
I think that this would be more feasible on macOS due to the fact that multiple windows exist under a single application. Otherwise, one would need to implement some kind of split screen feature or IPC to actually be able to communicate different zoom levels.

just a thought as i review all open issues here today

**Comment by jurplel** [comment]
Oh yeah, I literally already wrote that.

great

**Comment by porg** [comment]
Yes. And the Mac app with this feature would still be appreciated! Still subscribed and interested.

**Comment by jurplel** [comment]
This would definitely be a cool feature, but I don't think it will be worked on soon since the time I have to dedicate to the project is small + I want to avoid feature creep
